packages/network-of-terms-catalog/catalog/datasets/rijksmonumenten.jsonld

```diff
@@ -0,0 +1,64 @@
+{
+    "@context": "https://schema.org/docs/jsonldcontext.jsonld",
+    "@id": "https://linkeddata.cultureelerfgoed.nl/cho-kennis/id/rijksmonument/",
+    "@type": "Dataset",
+    "name": [
+      {
+        "@language": "nl",
+        "@value": "Rijksmonumenten"
+      }
+    ],
+    "alternateName": [
+      {
+        "@language": "nl",
+        "@value": "Rijksmonumentenregister"
+      }
+    ],
+    "creator": [
+      {
+        "@id": "https://www.cultureelerfgoed.nl"
+      }
+    ],
+    "url": [
+      "https://linkeddata.cultureelerfgoed.nl/cho-kennis/id/rijksmonument/"
+    ],
+    "mainEntityOfPage": [
+      "https://linkeddata.cultureelerfgoed.nl/rce/Rijksmonument"
+    ],
```

Comment:
@rschalkrce I had to add this property to make the description valid. Please let me know if a better landing page for this dataset is available.